### PR TITLE
Add Warsaw public_transport_tickets vending machine operators

### PR DESCRIPTION
**Pull request by docentYT**
[Source](https://wiki.openstreetmap.org/wiki/Pl:Tag:vending%3Dpublic_transport_tickets) (`ZTM Warszawa` header)
This is how all vending machines in the Polish capital are tagged.

### CODE REVIEW

File: data/operators/amenity/vending_machine.json [hunk 1]
```diff
@@ -29,6 +29,67 @@
...
+      "locationSet": {"include": ["pl"]},
+      "tags": {
+        "amenity": "vending_machine",
+        "vending": "public_transport_tickets",
+        "operator": "Mera-Serwis SGL",
+        "network": "network=ZTM Warszawa",
+        "network:wikidata": "Q8066998",
+        "network:wikipedia": "pl:Zarząd Transportu Miejskiego w Warszawie",
+        "currency:PLN": "yes",
+        "payment:coins": "yes",
```
Comment (LaoshuBaby) [inline data/operators/amenity/vending_machine.json:44]: I'm troubled with payment related tag

See https://github.com/osmlab/name-suggestion-index/wiki/Judge-Case

Some times we allow the addition of payment, but more time we reject it. @bhousel 

---
In my opinion, NSI shouldn't be a treasure chest with every kind of detailed tags, those should be added after on ground survey. But if this entry belongs to a public transport, we have enough reason believe that every single machine support same payment.

（> <）！

**Review by arch0345 — CHANGES_REQUESTED** [review]
Please note that we no longer include `:wikipedia` tags in NSI (see https://github.com/osmlab/name-suggestion-index/wiki/Judge-Case#can-we-add-websitephonewikipedia)

File: data/operators/amenity/vending_machine.json (hunk 1 above)
Comment (docentYT) [reply data/operators/amenity/vending_machine.json:43]: In Warsaw, single-operator vending machines accept the same forms of payment.

File: data/operators/amenity/vending_machine.json [hunk 2]
```diff
@@ -29,6 +29,64 @@
...
       }
+    },
+    {
+      "displayName": "Mera-Serwis SGL (ZTM Warszawa)",
+      "locationSet": {"include": ["pl"]},
+      "tags": {
+        "amenity": "vending_machine",
+        "vending": "public_transport_tickets",
+        "operator": "Mera-Serwis SGL",
+        "network": "network=ZTM Warszawa",
```
Comment (bhousel) [inline data/operators/amenity/vending_machine.json:40]: This looks like a typo?
```suggestion
        "network": "ZTM Warszawa",
```

Comment (docentYT) [reply data/operators/amenity/vending_machine.json]: Yes, thank you!

**Comment by LaoshuBaby** [comment]
Wait, I just found another problem……

@docentYT You add `network:wikidata` for "ZTM Warszawa" in 3 entries, but they are in `data/operator/amenity` tree, so if you want them be suggested, you'd better complete `operator:wikidata` for them.

**Comment by docentYT** [comment]
@LaoshuBaby but `network:wikidata` for ZTM Warszawa is the correct tagging for Warsaw's ticket machines. The company operating the machines is listed as an operator.

**Comment by LaoshuBaby** [comment]
> @LaoshuBaby but `network:wikidata` for ZTM Warszawa is the correct tagging for Warsaw's ticket machines. The company operating the machines is listed as an operator.

I don't means change it or remove it, but add wikidata as operator at the same time. Like this:

```
brand=1
brand:wikidata=Q12345678
network=ZTM Warszawa
network:wikidata=Q87654321
```

Do you think this is accaptable or this way can tag those vending machine correctly?

**Comment by bhousel** [comment]
Yeah these vending machines can be in the `data/operators/amenity/vending_machine.json` file, but they really need the `operator`/`operator:wikidata` tags existing, otherwise nothing will happen.

The `network`/`network:wikidata` tag is fine - when in the operators tree these are ignored (treated like any other random tags).

**Comment by docentYT** [comment]
@LaoshuBaby @bhousel I've just added operator:wikidata.

**Review by arch0345 — APPROVED** [review]

**Pull request merged**